config.json

```diff
@@ -29,6 +29,7 @@
 		"description": "meta[name=\"description\"]",
 		"text": "h1, h2, h3, h4, h5, h6, p, li"
 	},
+	"trimSize": 10000,
```

Comment:
I think we should name this `maxRecordSize`. Also, can you add a description of the option in the read me please ?
Thanks a lot for your contribution !